.github/workflows/test.yml

```diff
@@ -5,9 +5,8 @@ jobs:
   build:
     strategy:
       matrix:
-        # Keep the matrix for when jdk 17 rc come out
-        java: [17, 18]
-    runs-on: ubuntu-20.04
+        java: [17, 19]
```

Comment:
I'd keep it at 18 until a new Gradle release supporting Java 19 appears, since yeah, gradle's kinda the dealbreaker of the newer Java versions and ughhhhh i hate that so much, but unfortunately, that's life

Reply:
Even then, I would just add 19 as a separate item.

Reply:
Undoing that then.

Reply:
meh, I would drop Java 18 as soon as possible